apps/module-scan/src/cli/retry-scan/index.test.ts

```diff
@@ -0,0 +1,300 @@
+import { tmpNameSync } from 'tmp'
+import Store from '../../store'
+import { retryScan, queryFromOptions } from './'
+import { parseOptions } from './options'
+import * as fixtures from '../../../test/fixtures/state-of-hamilton'
+
+if (process.env.CI) {
+  jest.setTimeout(10000)
```

Comment:
What happens if this times out, does circleci fail?

Reply:
Yes.